### PR TITLE
Fix typos identified by IntelliJ

### PR DESCRIPTION
**Pull request by JackPGreen**
Fixes https://github.com/hazelcast/cloud-docs/issues/117

### CODE REVIEW

File: docs/modules/ROOT/pages/connect-to-cluster.adoc [hunk 1]
```diff
@@ -47,7 +47,7 @@ include::partial$get-connection-creds.adoc[]
 [[sql]]
 == Using the SQL Browser
 
-You can open a SQL shell in your cluster and submit SQL queries directly from your web browser.
+You can open an SQL shell in your cluster and submit SQL queries directly from your web browser.
```
Comment (angelasimms) [inline docs/modules/ROOT/pages/connect-to-cluster.adoc:50]: This is not correct even though IntelliJ might think so.

File: docs/modules/ROOT/pages/create-account.adoc [hunk 2]
```diff
@@ -63,7 +63,7 @@ After you set up your account, you'll want to keep it safe. Here are some recomm
 
 - *Keep private information secure:* Keep your password and any xref:developer.adoc[API keys] private.
 
-- *Enable multi-factor authentication:* Protect your account from anyone who may know your username and password. With xref:multi-factor-authentication.adoc[multi-factor authentication], you'll need to provide a unique code that's generated on your device as well as your username and password to sign in.
+- *Enable multifactor authentication:* Protect your account from anyone who may know your username and password. With xref:multi-factor-authentication.adoc[multifactor authentication], you'll need to provide a unique code that's generated on your device as well as your username and password to sign in.
```
Comment (angelasimms) [inline docs/modules/ROOT/pages/create-account.adoc:66]: This is not incorrect, simply an editorial choice. We tend to follow Merriam Webster: https://www.merriam-webster.com/dictionary/multi-factor%20authentication

File: docs/modules/ROOT/pages/get-started.adoc [hunk 3]
```diff
@@ -185,13 +185,13 @@ You should see your installed version of the CLI client.
...
 
-. Execute the following command to connect the SQL shell to your cluster. You may need add a path to the hz-cli program.
+. Execute the following command to connect the SQL shell to your cluster. You may need to add a path to the hz-cli program.
 +
 ```bash
 hz-cli -f hazelcast-client-with-ssl.yml sql
 ```
 +
-A SQL prompt is displayed:
+An SQL prompt is displayed:
```
Comment (angelasimms) [inline docs/modules/ROOT/pages/get-started.adoc:194]: IntelliJ is incorrect I'm afraid.

File: docs/modules/ROOT/pages/get-started.adoc [hunk 4]
```diff
@@ -232,7 +232,7 @@ INSERT INTO cities VALUES
 [[step-3]]
 == Step 3. Query the Cache with SQL
 
-Now that you have some data in your cluster, you can query it, using SQL. If you're using the CLI, enter the following  queries in the SQL prompt. If you're using a client library, enter the following queries in the SQL browser.
+Now that you have some data in your cluster, you can query it using SQL. If you're using the CLI, enter the following  queries in the SQL prompt. If you're using a client library, enter the following queries in the SQL browser.
```
Comment (angelasimms) [inline docs/modules/ROOT/pages/get-started.adoc:235]: IntelliJ is incorrect again. Can you leave as is.

File: docs/modules/ROOT/pages/multi-factor-authentication.adoc [hunk 5]
```diff
@@ -1,5 +1,5 @@
-= Multi-Factor Authentication
-:description: Multi-factor authentication (MFA) makes your account more secure by adding an extra step to the sign-in process. Instead of just entering a password, MFA allows you to use a trusted device to generate a secure digital pin for authentication.
+= Multifactor Authentication
```
Comment (angelasimms) [inline docs/modules/ROOT/pages/multi-factor-authentication.adoc:1]: As per earlier comment. Can you leave as is?

File: docs/modules/ROOT/pages/multi-factor-authentication.adoc [hunk 6]
```diff
@@ -1,5 +1,5 @@
...
+:description: Multifactor authentication (MFA) makes your account more secure by adding an extra step to the sign-in process. Instead of just entering a password, MFA allows you to use a trusted device to generate a secure digital pin for authentication.
```
Comment (angelasimms) [inline docs/modules/ROOT/pages/multi-factor-authentication.adoc:2]: As per earlier comment, can you leave as is.

**Review by angelasimms — CHANGES_REQUESTED** [review]
Hi Jack, Thanks for doing this. Just a few comments. Luckily a lot of this content is legacy and will be rewritten soon.

**Comment by JackPGreen** [comment]
@angelasimms are you able to have a look over the requested changes, please?

File: docs/modules/ROOT/pages/create-account.adoc [hunk 7]
```diff
@@ -63,7 +63,7 @@ After you set up your account, you'll want to keep it safe. Here are some recomm
 
 - *Keep private information secure:* Keep your password and any xref:developer.adoc[API keys] private.
 
-- *Enable multi-factor authentication:* Protect your account from anyone who may know your username and password. With xref:multi-factor-authentication.adoc[multi-factor authentication], you'll need to provide a unique code that's generated on your device as well as your username and password to sign in.
+- *Enable multi-factor authentication:* Protect your account from anyone who may know your username and password. With xref:multi-factor-authentication.adoc[multifactor authentication], you'll need to provide a unique code that's generated on your device as well as your username and password to sign in.
```
Comment (angelasimms) [inline docs/modules/ROOT/pages/create-account.adoc:66]:
```suggestion
- *Enable multi-factor authentication:* Protect your account from anyone who may know your username and password. With xref:multi-factor-authentication.adoc[multi-factor authentication], you'll need to provide a unique code that's generated on your device as well as your username and password to sign in.
```
Makes consistent with another change below.

File: docs/modules/ROOT/pages/multi-factor-authentication.adoc [hunk 8]
```diff
@@ -1,4 +1,4 @@
-= Multi-Factor Authentication
+= Multi-factor Authentication
```
Comment (angelasimms) [inline docs/modules/ROOT/pages/multi-factor-authentication.adoc:1]:
```suggestion
= Multi-Factor Authentication
```
For title case, this is correct.

**Review by angelasimms — CHANGES_REQUESTED** [review]
Just a couple of updates.

**Review by angelasimms — APPROVED** [review]

**Pull request merged**